<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindowClass</class>
 <widget class="QMainWindow" name="CMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>玻璃面板缺陷检测_V1.0.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="CMainWindow.qrc">
    <normaloff>:/CMainWindow/Resources/app.ico</normaloff>:/CMainWindow/Resources/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="MyLabel" name="label_Image">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="MyLabel" name="label_Image_Right">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/CMainWindow/Resources/background.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Start"/>
   <addaction name="action_Stop"/>
   <addaction name="action_Setting"/>
   <addaction name="action_DataManager"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Start">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/start.png</normaloff>:/CMainWindow/Resources/start.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/stop.png</normaloff>:/CMainWindow/Resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/settings.png</normaloff>:/CMainWindow/Resources/settings.png</iconset>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </action>
  <action name="action_DataManager">
   <property name="icon">
    <iconset resource="CMainWindow.qrc">
     <normaloff>:/CMainWindow/Resources/data.png</normaloff>:/CMainWindow/Resources/data.png</iconset>
   </property>
   <property name="text">
    <string>数据管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
